clang/test/dpct/cub/blocklevel/blockradixsort.cu

```diff
@@ -0,0 +1 @@
+

```

Comment:
Let's revert this change, it's unnecessary.

Reply:
Yes I removed this temporarily to see clang format failure in source. Added thanks